crates/neon/src/lib.rs

```diff
@@ -98,3 +98,16 @@ pub use neon_macros::*;
 
 #[cfg(feature = "napi-6")]
 mod lifecycle;
+
+#[cfg(feature = "napi-8")]
+static MODULE_TAG: once_cell::sync::Lazy<crate::sys::TypeTag> = once_cell::sync::Lazy::new(|| {
+    let mut lower = [0; std::mem::size_of::<u64>()];
```

Comment:
Maybe add a comment to say we aren't using this 64 bits so for now it's just reserved for future use.

Reply:
I'll add this. FYI, we're using 64-bits and not the full 128-bits.